static/app/components/dropdownMenu/index.tsx

```diff
@@ -200,7 +200,7 @@ function DropdownMenu({
 
   function renderTrigger() {
     if (trigger) {
-      return trigger({...overlayTriggerProps, ...buttonProps}, isOpen);
+      return trigger({...buttonProps, ...overlayTriggerProps}, isOpen);
```

Comment:
This ordering matches the ordering below for the default dropdown button. With the old order, toggling a dropdown menu by clicking its trigger was not working.